Lib/Rabbit4000/BIOSLIB/MUTIL.LIB

```diff
@@ -93,7 +93,7 @@ __root void L_div();
 ;          BCDE  = Dividend / Divisor
 ;          HL'HL = Dividend % Divisor
 ;	Modulus is given same sign as dividend (numerator).
-
+   align even

```

Comment:
As in my other reviews, I'd like to see at least a short comment on these `align` statements that we're targeting even alignment for improved performance on 5000/6000 with 16-bit memory.

Reply:
Done